src/amber/support/file_encryptor.cr

```diff
@@ -0,0 +1,33 @@
+require "./message_encryptor"
+
+module Amber::Support
+  module FileEncryptor
+    def self.global_encryption_key
+      ENV["AMBER_ENCRYPTION_KEY"]? || begin
+        if File.exists?(".encryption_key")
+          File.open(".encryption_key").gets_to_end.to_s
+        else
+          nil
+        end
+      end
+    end
+
+    def self.read(filename : String, encryption_key = global_encryption_key)
```

Comment:
I would propose that secret_key makes more sense in the FileEncryptor module than Amber Amber module since it's only needed there. This module should be used everywhere encrypted files are written or read. 

It's important for testing purposes that global_encryption_key can change when called in case in case it's called from inside of CLI tests after it's already been loaded by crystal spec.

Reply:
Agreed. I was under the impression that `secret_key` and `secret_key_base` could be used interchangeably.